src/components/createElementComponent.tsx

```diff
@@ -131,6 +131,10 @@ const createElementComponent = (
         let newElement: stripeJs.StripeElement | null = null;
         if (checkoutSdk) {
           switch (type) {
+            // @ts-expect-error - `paymentForm` Element is not part of stripeJs.StripeElement yet
```

Comment:
necessary until https://github.com/stripe/stripe-js/pull/829 is merged and deployed